src/main/resources/defaultProperties/cucumber.properties

```diff
@@ -1,5 +1,5 @@
 cucumber.filter.tags=
 cucumber.features=src/test/resources
 cucumber.extraGlue=
-cucumber.plugin=pretty, html:allure-results/cucumberReport.html, com.shaft.tools.listeners.CucumberFeatureListener
+cucumber.plugin=pretty, json:target/cucumber/Destination/cucumber.json, html:allure-results/cucumberReport.html, com.shaft.tools.listeners.CucumberFeatureListener
```

Comment:
Can we have the same path for the json similar to the html? Or do you want it by design to be in the target folder so that it gets deleted after every test run?
If you want it in the target folder, can we move it directly under "target/cucumber/cucumber.json" or what's the use of the "Destination" subdirectory here?

Reply:
Yes, sure we can have the same path for the JSON. i updated it to `allure-results/cucumber.json` in `cucumber.properties` and in the `JiraXray.properties`. it gets overwritten regardless of the path.